Lib/ufo2ft/otfPostProcessor.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function, division, absolute_import, unicode_literals
+
+import tempfile
+
+from compreffor import Compreffor
+from fontTools.ttLib import TTFont
+
+
+class OTFPostProcessor(object):
+    """Does some post-processing operations on a compiled OpenType font, using
+    info from the source UFO where necessary.
+    """
+
+    def __init__(self, otf, ufo):
+        self.ufo = ufo
+        tmp_file = tempfile.NamedTemporaryFile()
+        tmp_path = tmp_file.name
+        otf.save(tmp_path)
```

Comment:
Are you sure you can make a TTFont from BytesIO?

Trying that I get:

```
Traceback (most recent call last):
  File "c:\users\admin\downloads\trufont\lib\trufont\windows\fontWindow.py", line 383, in exportFile
    self._font.export(path)
  File "c:\users\admin\downloads\trufont\lib\trufont\objects\defcon.py", line 127, in export
    otf = self.getRepresentation("TruFont.TTFont")
  File "c:\users\admin\downloads\defcon\lib\defcon\objects\base.py", line 337, in getRepresentation
    representation = factory["factory"](self, **kwargs)
  File "c:\users\admin\downloads\trufont\lib\trufont\representationFactories\openTypeFactory.py", line 5, in TTFontFactory
    otf = compileOTF(font)
  File "c:\users\admin\downloads\ufo2ft\lib\ufo2ft\__init__.py", line 23, in compileOTF
    postProcessor = OTFPostProcessor(otf, ufo)
  File "c:\users\admin\downloads\ufo2ft\lib\ufo2ft\otfPostProcessor.py", line 21, in __init__
    self.otf = TTFont(stream)
  File "c:\users\admin\downloads\fonttools\lib\fontTools\ttLib\__init__.py", line 180, in __init__
    self.reader = sfnt.SFNTReader(file, checkChecksums, fontNumber=fontNumber)
  File "c:\users\admin\downloads\fonttools\lib\fontTools\ttLib\sfnt.py", line 84, in __init__
    raise ttLib.TTLibError("Not a TrueType or OpenType font (not enough data)")
fontTools.ttLib.TTLibError: Not a TrueType or OpenType font (not enough data)
```

and the code seems to expect a file path or file object in read mode.